src/utils/detect-server.js

```diff
@@ -22,19 +23,10 @@ module.exports.serverSettings = async (devConfig, flags, projectDir, log) => {
       }
     })
   } else if (devConfig.framework === '#auto' && !(devConfig.command && devConfig.targetPort)) {
-    const settingsArr = []
-    const detectors = detectorsFiles.map(det => {
-      try {
-        return loadDetector(det)
-      } catch (err) {
-        console.error(err)
-        return null
-      }
-    })
-    for (const detector of detectors) {
-      const detectorResult = detector(projectDir)
```

Comment:
Not sure why we were passing `projectDir` to `detector`

Reply:
This was dead code (not used in any detector).